packages/salesforce-adapter/src/constants.ts

```diff
@@ -164,6 +165,17 @@ export const FLOW_FIELD_TYPE_NAMES = {
   FLOW_STAGE_STEP_ENTRY_ACTION_OUTPUT_PARAMETER: 'FlowStageStepEntryActionOutputParameter',
 } as const
 
+export const SUPPORTED_ELEMENT_REFERENCE_PARENT_FIELDS = [
```

Comment:
```suggestion
export const FLOW_ELEMENT_REFERENCE_PARENT_FIELDS = [
```
Let's also add a comment that explains how to add support for additional references